packages/gen-ai-hub/src/client/openai/openai-client.test.ts

```diff
@@ -86,7 +84,7 @@ describe('openai client', () => {
       );
 
       await expect(
-        client.chatCompletion('gpt-4', prompt, '1234')
+        client.chatCompletion(prompt, { deploymentId: '1234' })
```

Comment:
[pp] I'd feel that
```suggestion
        client.chatCompletion({ deploymentId: '1234' }, prompt)
```
is a bit more intuitive, especially if we are adding `new OpenAiClient('gpt-4o').chatCompletion(prompt)`

Reply:
I think I agree, no strong opinion though. When I didn't it somehow felt wrong. The only thing that makes this better though, is that in orchestration it is an optional parameter and it couldn't be optional if it came first.

Reply:
@MatKuhr I think at this point in time, this was a useless change. I can revert it, but I would suggest to keep it and then fix it with the next PR where we introduce something of this kind: `new OpenAiClient('gpt-4o').chatCompletion(prompt)`.

Reply:
Why use the type in orchestration at all? Because there the model is a mandatory part of the payload..

But agreed, let's move that to a potential PR for `new OpenAiClient('gpt-4o').chatCompletion(prompt)` 😉

Reply:
In orchestration it is a bit different, because users could potentially pass the deployment ID in addition to the mandatory payload (as of now). I will also note this down in my proposals for the follow up.